python/lsst/pipe/supertask/activator.py

```diff
@@ -292,7 +321,11 @@ def load_super_task(super_taskname):
 
         if super_task_module:
             # Need to add a try -- except here
-            python_mod_stask = importlib.import_module(package.__name__ + '.' + super_task_module)
+            if hasattr(package, 'mock'):
+                (file_path, filename, data) = imp.find_module(super_task_module, package.__path__)
+                python_mod_stask = imp.load_module(super_task_module, file_path, filename, data)
```

Comment:
Ok, will do. It is not completely clear  to what extend this activator will be used to launch several jobs or under which circumstances  but that's definitely in the long term. I'll use `sys.path`